auditwheel/policy/__init__.py

```diff
@@ -26,12 +26,11 @@
     'manylinux2010_i686': ['linux_i686'],
 }
 
-# XXX: this could be weakened. The show command _could_ run on OS X or
-# Windows probably, but there's not much reason to inspect foreign package
-# that won't run on the platform.
 if platform != 'linux':
-    logger.critical('Error: This tool only supports Linux')
-    sys.exit(1)
+    logger.warning(f'Running on {platform}. Note that this tool only '
```

Comment:
I'm no expert on this but it seems the syntax used for logging here is only supported on python 3.6+ but not python 3.5
Can you update this so that tests do pass please ?

Reply:
Oops, sorry. I forgot that 3.5 is supported. Updated.